tests/fixtures.py

```diff
@@ -84,7 +84,9 @@ def local_device_factory(coolorg):
     count = 0
 
     def _local_device_factory(
-        base_device_id: Optional[str] = None, org: OrganizationFullData = coolorg
+        base_device_id: Optional[str] = None,
+        org: OrganizationFullData = coolorg,
+        is_admin: bool = False,
```

Comment:
Plutôt que de le mettre à `False` par défaut, je le mettrais à `None`, ensuite:
- si le user n'existe pas, je fais `is_admin = bool(is_admin)`
- si le user existe déjà, je raise une erreur si `is_admin is not None`

L'intérêt étant d'éviter qu'un code qui s'exécute dans cet ordre créé un user non admin sans qu'on s'en rende compte:
```python
local_device_factory('alice@dev2')
local_device_factory('alice@dev1', is_admin=True)
```